modules/loader-utils/src/lib/files/http-file.ts

```diff
@@ -30,8 +30,8 @@ export class HttpFile implements ReadableFile {
     };
   }
 
-  async read(offset: number | bigint, length: number): Promise<ArrayBuffer> {
-    const response = await this.fetchRange(offset, length);
+  async read(offset?: number | bigint, length?: number): Promise<ArrayBuffer> {
+    const response = await this.fetchRange(offset ?? 0, length ?? 0);
```

Comment:
Use default argument syntax for better clarity?
```suggestion
  async read(offset: number | bigint = 0, length: number = 0): Promise<ArrayBuffer> {
    const response = await this.fetchRange(offset, length);
```